src/components/maincontroller.ts

```diff
@@ -364,81 +376,44 @@ export function seek(ticks: number): Promise<void> {
     return changeStream(ticks);
 }
 
-export function changeStream(
+export async function changeStream(
     ticks: number,
     params: any = undefined
 ): Promise<void> {
     if (
-        window.mediaManager.getMediaInformation().customData.canClientSeek &&
+        window.playerManager.getMediaInformation().customData.canClientSeek &&
         params == null
     ) {
-        window.mediaManager.seek(ticks / 10000000);
+        window.playerManager.seek(ticks / 10000000);
         reportPlaybackProgress($scope, getReportingParams($scope));
-        return Promise.resolve();
+        return;
     }
 
     params = params || {};
 
-    const playSessionId = $scope.playSessionId;
-    const liveStreamId = $scope.liveStreamId;
-
-    const item = $scope.item;
-
-    return getMaxBitrate().then(async (maxBitrate) => {
-        const deviceProfile = getDeviceProfile({
-            enableHls: true,
-            bitrateSetting: maxBitrate
-        });
-        const audioStreamIndex =
+    // TODO Could be useful for garbage collection.
+    //      It needs to predict if the server side transcode needs
+    //      to restart.
+    //      Possibility: Always assume it will. Downside: VTT subs switching doesn't
+    //      need to restart the transcode.
+    //const requiresStoppingTranscoding = false;
+    //if (requiresStoppingTranscoding) {
+    //    window.playerManager.pause();
+    //    await stopActiveEncodings($scope.playSessionId);
+    //}
```

Comment:
We want those comments as well?

Reply:
Well, it's a WIP. I don't know yet what eventually needs to happen in this stream change and on what condition we have to ask the player to pause. I think it could be useful.